resources/js/Mixins/Global.js

```diff
@@ -113,6 +113,15 @@ export default {
         pluralize(value, number) {
             return pluralize(value, number);
         },
+
+        /*Extract Doi from URL*/
+        extractDoi(query) {
+            query = query.trim();
+            if (query.indexOf("https://doi.org/") > -1) {
+                query = query.substr(16);
+            }
+            return query;
+        },
```

Comment:
Use URL module to extract path and then sanitise the DOI before updating the input based on onblur event

Reply:
```
        extractDoi(query) {
            if(query.indexOf("http") > -1){
                var url = new URL(query);
                query = (url.pathname).replace('/','');
            }
            return query.trim();
        },
```